pages/instructorAssessmentInstance/instructorAssessmentInstance.sql

```diff
@@ -57,13 +57,22 @@ SELECT
     -- PostgreSQL so it can be properly interpreted when a grade
     -- update POST is received back.
     CAST(iq.modified_at AS TEXT) AS modified_at,
+    -- If auto_points and manual_points are null, these columns were
+    -- not updated since the implementation of separate manual and
+    -- auto points. This computation, in practice, assigns
+    -- manual_points if the question is manually graded or if the
+    -- points are above max_auto_points.
+    COALESCE(iq.auto_points, LEAST(iq.points, aq.max_auto_points)) AS auto_points,
+    COALESCE(iq.manual_points, GREATEST(0, iq.points - aq.max_auto_points)) AS manual_points,
     ((lag(z.id) OVER w) IS DISTINCT FROM z.id) AS start_new_zone,
     z.id AS zone_id,
     z.title AS zone_title,
     q.title AS question_title,
     q.id AS question_id,
     q.qid,
     aq.max_points,
+    aq.manual_points AS max_manual_points,
```

Comment:
I'm finding the renaming here (and similar renaming elsewhere) to be a bit confusing. I _think_ I understand why we are doing it. Would it be cleaner to rename `aq.manual_points` to `aq.max_manual_points` to match `aq.max_auto_points`? This would match the naming in `question_score_logs`.

Reply:
That's a great point. I'll rename it. Should I also rename it in the schema for infoAssessment.json (`maxManualPoints` instead of `manualPoints`)? I don't think it's necessary, as I imagine most instructors would not use `maxAutoPoints` often and having `manualPoints` matches the `autoPoints` field better.

Reply:
I agree with you that it's probably best to keep the input variable as `manualPoints` in `infoAssessment.json`.

Reply:
Resolved in ae83825.